include/srf/memory/resources/host/malloc_memory_resource.hpp

```diff
@@ -21,59 +21,22 @@
 
 namespace srf::memory {
 
-class malloc_memory_resource : public memory_resource<cuda::memory_kind::host>
+class malloc_memory_resource final : public memory_resource
```

Comment:
it doesn't support `memory_kind`, but it could be used as as base.

again, this is all temporary until RMM  / libcudacxx provides the generalized `memory_resource`